com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -114,22 +118,22 @@ static void DrawConfinerGizmos(CinemachineConfiner2D confiner2D, GizmoType type)
             if (!confiner2D.GetGizmoPaths(out var originalPath, ref s_CurrentPathCache, out var pathLocalToWorld))
                 return;
 
-            var color = CinemachineCorePrefs.BoundaryObjectGizmoColour.Value;
-            var colorDimmed = new Color(color.r, color.g, color.b, color.a / 2f);
-            
+            var pathCacheColor = CinemachineCorePrefs.BoundaryObjectGizmoColour.Value;
+            var inputColliderColor = new Color(145f / 255f, 244f / 255f, 139f / 255f, 1f);
```

Comment:
I don't like this hardcoded color.  It's better to make it a computed variant of the other color.  I still think the input polygon should be brighter than the computed one.  What was wrong with the colors before?